pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -482,9 +482,10 @@ type ServiceInstance struct {
 //
 // Currently supported ways:
 //  - ClusterServiceClassExternalName and ClusterServicePlanExternalName
+//  - ClusterServiceClassExternalID and ClusterServicePlanExternalID
 //  - ClusterServiceClassName and ClusterServicePlanName
 //
-// For both of these ways, if a ClusterServiceClass only has one plan
+// For any of these ways, if a ClusterServiceClass only has one plan
 // then leaving the *ServicePlanName is optional.
```

Comment:
Does this comment count for the plan ID like plan name?

Reply:
Yes thanks for noticing that, I'll update the comment.

Reply:
All fixed!